dll/cpl/joy/joy.c

```diff
@@ -115,9 +124,43 @@ CustomPageProc(HWND hwndDlg, UINT uMsg, WPARAM wParam, LPARAM lParam)
 {
     UNREFERENCED_PARAMETER(lParam);
 
+    WCHAR szBuf[2];
+    HWND dwComboHwnd;
+
     switch (uMsg)
     {
         case WM_INITDIALOG:
+            szBuf[1]=UNICODE_NULL;
+
+            CheckDlgButton(hwndDlg, IDC_JOYSTICK_RADIO, BST_CHECKED);
+
+            dwComboHwnd = GetDlgItem(hwndDlg,IDC_AXES_COMBO);
+            if (dwComboHwnd)
+            {
+                szBuf[0]=L'2';
+                SendMessageW(dwComboHwnd, CB_ADDSTRING, -1, (LPARAM)szBuf);
```

Comment:
What's this -1? It can be zero.

Reply:
Yes (copy / paste from another ROS module where -1 was used. Likely to be cleanup up there someday)